dataverse-webapp/src/main/webapp/dataverseuser.xhtml

```diff
@@ -675,6 +675,31 @@
                                         <p:message for="position" display="text" />
                                     </div>
                                 </div>
+                                <div class="form-group" jsf:rendered="#{DataverseUserPage.editMode == 'CREATE' or DataverseUserPage.editMode == 'EDIT'}">
+                                    <!-- E-mail notifications language -->
+                                    <label for="position" class="col-sm-3 control-label">
+                                        #{bundle['user.notificationsLanguage']}
+                                        <span class="glyphicon glyphicon-question-sign tooltip-icon"
+                                              data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['user.notificationsLanguage.tip']}"></span>
+                                    </label>
+                                    <div class="col-sm-4">
+                                        <p:selectOneMenu styleClass="facet-category-default"
+                                                         disabled="#{false}"
+                                                         label="#{DataverseUserPage.getLocalizedPreferredNotificationsLanguage()}"
+                                                         value="#{DataverseUserPage.preferredNotificationsLanguage}"
+                                        >
+                                            <f:selectItem
+                                                    itemLabel="#{bundle['user.notificationsLanguage.selectItem']}"
```

Comment:
I wonder if this is needed, it's kinda weird that when you select a language there is some greyed out option. If there is no option to set some "template" option that does not appear on the list, I would just leave english as default.

Reply:
By default we have sessions language. For @wfenrich to decide.